test/Microsoft.ML.Tests/ImagesTests.cs

```diff
@@ -200,9 +200,9 @@ public void TestBackAndForthConversionWithAlphaInterleave()
             var images = new ImageLoadingTransformer(env, imageFolder, ("ImageReal", "ImagePath")).Transform(data);
             var cropped = new ImageResizingTransformer(env, "ImageCropped", imageWidth, imageHeight, "ImageReal").Transform(images);
 
-            var pixels = new ImagePixelExtractingTransformer(env, "ImagePixels", "ImageCropped", ImagePixelExtractingEstimator.ColorBits.All, true, 2f / 255, 127.5f).Transform(cropped);
-            IDataView backToBitmaps = new VectorToImageConvertingTransformer(env, "ImageRestored", "ImagePixels",
-                imageHeight, imageWidth, ImagePixelExtractingEstimator.ColorBits.All, true, 255f / 2, -1f).Transform(pixels);
+            var pixels = new ImagePixelExtractingTransformer(env, "ImagePixels", "ImageCropped", ImagePixelExtractingEstimator.ColorBits.All, interleave: true, scale: 2f/19, offset: 30).Transform(cropped);
```

Comment:
>cale: 2f/19, offset: 30) [](start = 161, length = 24)

why are the scale and offset changing?

Reply:
It was hard to test. all first pixels are white which is 255 in each channel. (255-127.5)*(2/255) = 1.
And I didn't like old vector to image logic of scaling.
which was (pixel -offset)*scale. 
Since this two transforms are tangle up together i prefer scale in PixelExtraction would be 1/scale in VectorToImage, and offset = -offset.
This way they opposite.

---
In reply to: [259550473](https://github.com/dotnet/machinelearning/pull/2602#discussion_r259550473) [](ancestors = 259550473)